prov/sockets/src/sock_ep_msg.c

```diff
@@ -710,6 +712,7 @@ static int sock_ep_cm_connect(struct fid_ep *ep, const void *addr,
 	sock_fd = ofi_socket(AF_INET, SOCK_STREAM, 0);
 	if (sock_fd < 0) {
 		SOCK_LOG_ERROR("no socket\n");
+		ret = -errno;
```

Comment:
I think this needs to use ofi_sockerr() instead of errno directly to handle windows correctly